zeebe/atomix/cluster/src/test/java/io/atomix/raft/ControllableRaftContexts.java

```diff
@@ -428,10 +414,9 @@ public void restartWithDataLoss(final MemberId memberId) {
       newContext.getCluster().bootstrap(raftServers.keySet());
     }
 
-    raftServers.put(memberId, newContext);
```

Comment:
This is not needed since the newContext is added to raftServers  via `createRaftContextForMember`